### DISCUSSION TITLE
Automatic and Arbitrary Geo Location blacklisting

### DISCUSSION THREAD

**Discussion by MigHubSolutions** [vercel/next.js#62839] [Help]
### Summary

I successfully deployed my company web App and everything was working fine. until yesterday which was on the 3rd of March 2024 my clients from website Africa were not able to access the website their Geo Location has been automatically blacklisted. Clients from Europe and America could make use of the web App, and people from East and North Africa could access the web app just people from Nigeria and Ghana. what is the cost of this issue? I didn't write any Geo Location blocking function and it was always working fine for all locations until Sunday.

### Example

www.mighub.travel

**Comment by Iyin0** [comment]
I encountered the same issue and noticed on 6th March 2024 for two websites: www.leytro.com and www.quickly.com.ng

They are still not going. I am in Nigeria

**Comment by luci53** [comment]
not that i am a pro but did you guys checked  your server's firewall and security rules to make sure there are no unintentional blocks for specific IP ranges or regions.

**Reply by MigHubSolutions** [reply]
No there was none. But please can elaborate where this settings with a screenshot

**Reply by Iyin0** [reply]
There was no blocks that was done, the deployment settings was not even touched since deployment. The websites were working fine until 2 days ago

**Reply by luci53** [reply]
sometimes they do that automatically try to have  a word with host or check there announcments or something because other than hosting as you guys are saying there should be no issue in other stuff

**Reply by Iyin0** [reply]
I have a couple of subdomains and they are going through. The issue only lies with the main domain. 

@MigHubSolutions the link you posted is going from here (Nigeria). Did you make any adjustments?

**Comment by MigHubSolutions** [comment]
The problem is the main domain misbehaves some people can access the website now whereas some can't access the website and some of the people that were able to access the website this morning from Nigeria cant access it again now, i cleared the cache and date of the deployment and thought the problem would be solved but no it wasn't been resolved. I believe there is an issue with the Vercel server configuration that is either unnoticed or unresolved yet

**Comment by mohemos** [comment]
I am facing the same issue with monirates.com people from other countries in the world can access it except Nigeria

**Comment by aljalcantara** [comment]
Im facing the same issue with a website I deployed. Our clients from West Africa cant access the website. Here's the website https://lion-share.io

**Reply by Iyin0** [reply]
Make the `www.` subdomain the main domain and re-route from `lion-share.io` to `www.lion-share.io`

**Comment by raymondmbam** [comment]
Is there any update on this?
